test/new-e2e/pkg/environments/aws/docker/host.go

```diff
@@ -109,80 +119,86 @@ func WithoutAgent() ProvisionerOption {
 	}
 }
 
-// Provisioner creates a VM environment with an EC2 VM with Docker, an ECS Fargate FakeIntake and a Docker Agent configured to talk to each other.
-// FakeIntake and Agent creation can be deactivated by using [WithoutFakeIntake] and [WithoutAgent] options.
-func Provisioner(opts ...ProvisionerOption) e2e.TypedProvisioner[environments.DockerHost] {
-	// We need to build params here to be able to use params.name in the provisioner name
-	params := newProvisionerParams()
-	err := optional.ApplyOptions(params, opts)
-	if err != nil {
-		panic(fmt.Errorf("unable to apply ProvisionerOption, err: %w", err))
-	}
-
-	provisioner := e2e.NewTypedPulumiProvisioner(provisionerBaseID+params.name, func(ctx *pulumi.Context, env *environments.DockerHost) error {
-		// We ALWAYS need to make a deep copy of `params`, as the provisioner can be called multiple times.
-		// and it's easy to forget about it, leading to hard to debug issues.
-		params := newProvisionerParams()
-		_ = optional.ApplyOptions(params, opts)
+type ProvisionerPulumiEnv struct {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Do we still need this ? I cannot find usages of this in the PR

Reply:
nope thanks